src/views/product-tutorials-view/components/product-view-content/index.tsx

```diff
@@ -18,8 +18,6 @@ const SUPPORTED_BLOCK_TYPES = [
 
 function ProductViewContent({
 	blocks,
-	inlineCollections,
-	inlineTutorials,
```

Comment:
No longer any need to pass `inlineCollections` and `inlineTutorials` to the client, as anywhere they were previously needed, we're now doing the data formatting for components server-side.